templates/includes/faq.html

```diff
@@ -6,10 +6,21 @@
       <h2 class="grid_separator">Frequently Asked Questions</h2>
 
       <h4 class="grid_separator--s">What is a sustaining membership?</h4>
-      <div class="prose grid_separator--l"><p>A sustaining membership means your donation recurs automatically on a monthly or yearly basis via credit card. This provides the Tribune with the sort of predictable revenue that allows us to plan budgets a few years at a time, and it helps eliminate renewal mailings or lapses in your membership.</p></div>
+      <div class="prose grid_separator--l"><p>
+        A sustaining membership means your donation recurs automatically on a monthly or yearly basis via credit card. This provides the Tribune with the sort of predictable revenue that allows us to plan budgets a few years at a time, and it helps eliminate renewal mailings or lapses in your membership.
+
+        </p> You can update the credit card information associated with your monthly or yearly donation on your Tribune account. <a href="http://texastribune.org/login">Log in</a>, then update your payment information from the “Membership” tab.</p>
+
+        </p>Don’t have a Tribune account? <a href="http://texastribune.org/signup">Sign up</a> to view your donation history and  membership status, update card information for recurring donations, and download your yearly tax receipt. <a href="https://www.texastribune.org/account-faq">Learn more about Tribune accounts. </a></p>
+
+      </p></div>
 
       <h4 class="grid_separator--s">How can I renew my membership?</h4>
-      <div class="prose grid_separator--l"><p>Sustaining memberships automatically renew on a monthly or yearly basis. If you need to update your billing information, or your membership is up for renewal soon, please contact <a href="mailto:membership@texastribune.org">membership@texastribune.org</a>.</p></div>
+      <div class="prose grid_separator--l">
+        <p>If you’re looking to renew your membership, make a donation to the Tribune. Your membership renews for one full year after you make a donation.</p>
+
+        <p>Sustaining memberships automatically renew on a monthly or yearly basis. If you need to update your credit card information, <a href="http://texastribune.org/login">log in</a> to your Tribune account. If you need assistance, please contact <a href="mailto:membership@texastribune.org">membership@texastribune.org</a>.</p>
```

Comment:
This url will auto redirect to https:// but we should reference `https://texastribune.org/login` instead for good measure. (I know that's probably just how it was in Airtable 😄)

Reply:
Thank you for this. I am assuming you meant for https://texastribune.org/signup . I am updating it to /login. Yep, I was following the airtable. Very good to know about this redirect.

Thank you for looking at it so closely.